bin/run_obs_yul_all.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Please add a comment explaining what this file does.

Reply:
got it!